src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -368,6 +373,12 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         }
     }
 
+    private onAssetError = () => {
```

Comment:
What is the goal here? To persisted that the asset has an error?  Seems unnecessary.